app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,10 @@
+class CustomersController < ApplicationController
+  def show
+    @customer = Customer.find(params[:id])
+  end
+
+  def update
```

Comment:
We would probably want to not create an `update` here in our `customers` controller when dealing with our `CustomerItem` relationship. The reason being is that if we needed to make an update on a `customer` we no longer have that CRUD action available to us. So instead, we would want to create a `customer_items_controller` with a `create` action since we are technically creating a new association between the two.